src/test/java/sanghwa/CalculatorTest.java

```diff
@@ -0,0 +1,60 @@
+package sanghwa;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+//한글 메서드명 대신 @DisplayName으로 쓰자
+@DisplayName("calculator tester")
+class CalculatorTest {
+    private final Calculator calculator = new Calculator();
+
+    //@Nested 중첩 클래스 선언, 여러가지 메서드를 사용할 수 있음.
+    @Nested
+    @DisplayName("덧셈")
+    class Add {
+        @Test
+        @DisplayName("일반 덧셈 테스트")
+        void add() {
+            assertEquals(5, calculator.add(2, 3));
+        }
+    }
+
+    @Nested
+    @DisplayName("뺄셈")
+    class Subtract {
+        @Test
+        @DisplayName("일반 뺄셈 테스트")
+        void subtract() {
+            assertEquals(-3, calculator.subtract(3, 6));
+        }
+    }
```

Comment:
아래 메서드들과 동일합니다
`displayName`은 상화님이 생각하기에 어떻게 사용되기를 의도한 어노테이션인 것 같나요?
클래스 이름만으로도 뺄샘의 뜻을 명확히 알 수 있지 않나요?

아래와 같이 작성되어도 해당 메서드가 테스트인가에 대한 의도 전달, 
어떤 동작을 테스트하는지에 대한 의도가 전달되는지 편하게 알 수 있지 않나요?
고민해보고 하은이의 생각을 알려주세요!
```suggestion
    @Nested
    class Subtract {
        @Test
        void subtract() {
            assertEquals(-3, calculator.subtract(3, 6));
        }
    }
```

Reply:
말씀해주신대로 이번 미션에서는 @DisplayName 을 쓰지 않아도 충분히 그 의도가 전달된다고 생각합니다.
다만, @DisplayName 과 친해지기 위해 의도적으로 사용해보고자 한 부분도 있었던 것 같습니다.

한 가지 살짝 고민했던 부분은, 코드의 일관성을 위해 @DisplayName 을 사용할꺼면 쭉 사용하는게 좋을지, 아니면 필요한 곳에만 군데군데 넣는 것이 좋을지 고민했었습니다. 하지만, 지금까지 쭉 리뷰를 달다보니 @DisplayName 도 주석처럼 필요할 때만 쓰는 것이 가장 좋다! 라는 결론을 내리게 됩니다.

코드를 필요할 때만 쓰는 습관을 더 길러야겠습니다. 리뷰 달아주시고 좋은 질문들 던져주셔서 감사합니다!